### PR TITLE
Rename to aleth

### PR DESCRIPTION
**Pull request by chfast**

### CODE REVIEW

File: doc/gitbook/README.md [hunk 1]
```diff
@@ -8,7 +8,6 @@ cpp-ethereum is a distribution of software including a number of diverse tools.
 The full software suite of cpp-ethereum includes:
 
 - **eth** (`eth`) The mainline CLI Ethereum client. Run it in the background and it will connect to the Ethereum network; you can mine, make transactions and inspect the blockchain.
```
Comment (gumb0) [inline doc/gitbook/README.md]: what about this line?

File: doc/gitbook/aleth-key.md [hunk 2]
```diff
@@ -256,23 +256,23 @@ All restored!
 
 Because our keys all share the same format it's really easy to import keys from other clients like Geth. In fact it's exactly the same process as restoring a key from a previous backup as we did in the last step.
 
-If we assume we have a geth key at `mygethkey.json`, then to import it to use `eth`, simply use:
```
Comment (gumb0) [inline doc/gitbook/aleth-key.md]: this is about importing keys from geth, it should stay `mygethkey.json`

**Comment by chfast** [comment]
Fixed.

File: doc/gitbook/aleth-key.md [hunk 3]
```diff
@@ -256,23 +256,23 @@ All restored!
...
+If we assume we have a geth key at `mygeth-key.json`, then to import it to use `aleth`, simply use:
 
 ```
-> ethkey import mygethkey.json "My Old Geth Key"
+> aleth-key import mygaleth-key.json "My Old Geth Key"
```
Comment (gumb0) [inline doc/gitbook/aleth-key.md]: mygeth-key.json

Comment (chfast) [reply doc/gitbook/aleth-key.md]: Fixed.

**Pull request merged**